README.md

```diff
@@ -8,12 +8,11 @@ The no-login variant is currently in alpha and is supposed to replace the [remot
 
 The UI is built using ReactJS and Redux.
 
-## Setup
+## Development setup
```

Comment:
We have "for development" and "for production" sections here so I'd leave "##Setup".